omod/src/main/webapp/pages/visit/templates/encounters/hivNotMigratedEncounterLong.gsp

```diff
@@ -0,0 +1,30 @@
+<div class="header" ng-include="'templates/encounters/defaultEncounterHeader.page'">
+</div>
+
+<div class="content encounter-summary-long">  <!-- encounter-summary-long currently only used for Selenium tests -->
+    <p class="bold">
+        ${ ui.message("pihcore.hiv.legacyDataNotMigrated") }
+    </p>
+    <p ng-repeat="obs in encounter.obs" class="aligned">
+        <label>{{ obs.concept | omrsDisplay }}</label>
+        <span class="value">{{ obs | obs:"value" }}</span>
```

Comment:
I don't know if this has any special functionality that isn't obvious here, but I'd think we'd want to try, at a minimum, to at least organize Obs by Obs Group if possible (display them together in some sort of field set maybe, with the grouping obs as the field set header).